spec/lib/stripe/payment_intent_validator_spec.rb

```diff
@@ -60,11 +60,22 @@
               expect(result).to eq payment_intent_response_body
             }.to_not raise_error Stripe::StripeError
           end
+
+          it "captures the payment" do
+            expect(Stripe::PaymentIntent.retrieve(
+              payment_intent.id
+            ).status).to eq("requires_capture")
+
+            Stripe::PaymentIntent.capture(payment_intent.id)
+
+            expect(Stripe::PaymentIntent.retrieve(
+              payment_intent.id
+            ).status).to eq("succeeded")
+          end
```

Comment:
This spec should be testing the Stripe::PaymentIntentValidator which is our class. The spec before this is calling the validator and checking that it doesn't raise errors but this test case is not calling it at all. As you said in your comment, it's testing Stripe directly and a similar test is probably performed by the Stripe team already. So we don't have to test this.